### DISCUSSION TITLE
Crear una comunidad

### DISCUSSION THREAD

**Discussion by gerryMartz** [Launch-X-Latam/MisionFrontEnd#86] [Q&A]
Hola devs!

Tengo una duda, ya que no podemos crear grupos por otros medios, ¿Cómo podremos generar una comunidad cuando esto termine si no podemos tener los medios de contacto de otras personas, como whatsapp, fb, discord, etc?

**Comment by doguedogue** [comment]
Aqui puedes seguir a las personas por Github, la comunidad tendrá la organización de Launch X Latam

Ademas normalmente en el perfil se puede publicar tu twitter o linkedin, es una buena forma también de seguir a los demás.

Saludos!!!

**Comment by OscarTinajero117** [comment]
ya existe la respuesta: https://github.com/Launch-X-Latam/MisionFrontEnd/discussions/2
